image/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG  BASE
+
+FROM --platform=linux/amd64 ubuntu:20.04 as linuxbase
```

Comment:
is needed if you are are not doing anything on this stage?

Reply:
It is not needed. I'll remove it.